phpstan.dist.neon

```diff
@@ -17,6 +17,9 @@ parameters:
         - '/^Class [a-zA-Z\\]+ extends @final class PhpCsFixer\\(ConfigurationException\\InvalidConfigurationException|ConfigurationException\\InvalidFixerConfigurationException|Tokenizer\\Tokens|Console\\Command\\FixCommand)\.$/'
         - '/^\$this\(PhpCsFixer\\Tokenizer\\Tokens\) does not accept PhpCsFixer\\Tokenizer\\Token\|null\.$/'
 
+        # We can't add these types because we support PHP 7.4-8.3 (which means different Symfony versions with incompatible Command signatures)
```

Comment:
This can be reverted and types in command can be add as in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7842.

Reply:
It was removed during rebase, I've added `@var` to those 2 properties just like you did in other commands 👍.